src/apps/app_net_crawler.erl

```diff
@@ -62,19 +62,27 @@ start_worker(Node) ->
 		end
 	).
 
-%% @doc Try to fetch peers for a node. Will timeout and try up to 3 times with
+%% @doc Try to fetch peers for a node. Will timeout and try up to 4 times with
 %% a 20 seconds delay in between.
 fetch_peers(Node) ->
-	fetch_peers(Node, 3).
+	fetch_peers(Node, {1, 4}).
 
-fetch_peers(_, 0) ->
+fetch_peers(_, {_Attempt, _MaxAttempts}) when _Attempt >= _MaxAttempts ->
 	unavailable;
-fetch_peers(Node, AttemptsLeft) ->
-	case ar_http_iface_client:get_peers(Node) of
+fetch_peers(Node, {Attempt, MaxAttempts}) when Attempt > 1 ->
+	timer:sleep(30 * 1000),
```

Comment:
I do not know if and how we use `app_net_crawler`, but anyway wonder why we set such big timeouts.